cpu/esp32/doc.txt

```diff
@@ -53,131 +53,140 @@
-[SPI1_CTRL](#esp32_spi_interfaces)  | SPI Controller used for SPI_DEV(1), can be ```VSPI``` ```HSPI``` | o
-[SPI1_SCK](#esp32_spi_interfaces)  | GPIO used as SCK for SPI_DEV(1)        | o
-[SPI1_MOSI](#esp32_spi_interfaces) | GPIO used as MOSI for SPI_DEV(1)       | o
-[SPI1_MISO](#esp32_spi_interfaces) | GPIO used as MISO for SPI_DEV(1)       | o
-[SPI1_CS0](#esp32_spi_interfaces)  | GPIO used as default CS for SPI_DEV(1) | o
-[UART1_TXD](#esp32_uart_interfaces) | GPIO used as TxD for UART_DEV(1) | o
-[UART1_RXD](#esp32_uart_interfaces) | GPIO used as RxD for UART_DEV(1) | o
-[UART2_TXD](#esp32_uart_interfaces) | GPIO used as TxD for UART_DEV(2) | o
-[UART2_RXD](#esp32_uart_interfaces) | GPIO used as RxD for UART_DEV(2) | o
-
-</center>
-
-<b>*Type:</b> m - mandatory, o - optional
-
-The following table gives a short reference  in alphabetical order of modules that can be enabled/disabled by board configurations and/or application's makefile using ```USEMODULE``` and ```DISABLE_MODULE```.
-
-<center>
-
-Module    | Default  | Short description
-----------|----------|-------------------
-[esp_eth](#esp32_ethernet_network_interface) | not used | enable the Ethernet MAC (EMAC) network device
-[esp_gdb](#esp32_debugging) | not used | enable the compilation with debug information for debugging
-[esp_hw_counter](#esp32_timers) | not used | use hardware counters for RIOT timers
-[esp_i2c_hw](#esp32_i2c_interfaces) | not used  | use the i2C hardware implementation
-[esp_idf_heap](#esp32_esp_idf_heap_implementation) | not used | enable ESP-IDF heap implementation
-[esp_log_colored](#esp32_esp_log_module) | not used | enable colored log output
-[esp_log_startup](#esp32_esp_log_module) | not used | enable additional startup information
-[esp_log_tagged](#esp32_esp_log_module) | not used | add additional information to the log output
-[esp_now](#esp32_esp_now_network_interface) | not used  | enable the ESP-NOW network device
-[esp_rtc_timer_32k](#esp32_rtt_counter) | not used | use RTC timer with external 32.768 kHz crystal as RTT
-[esp_spi_ram](#esp32_spi_ram) | not used |  enable SPI RAM
-[esp_spiffs](#esp32_spiffs_device) | not used  |  enable SPIFFS for on-board flash memory
-[esp_wifi](#esp32_wifi_network_interface) | not used  | enable the Wifi network device in WPA2 personal mode
-[esp_wifi_ap](#esp32_wifi_ap_network_interface) | not used | enable the WiFi SoftAP network device
-[esp_wifi_enterprise](#esp32_wifi_network_interface) | not used  | enable the Wifi network device in WPA2 enterprise mode
-
-</center>
-
-# <a name=esp32_mcu_esp32> MCU ESP32 </a> &nbsp;[[TOC](#esp32_toc)]
-
-ESP32 is a low-cost, ultra-low-power, single or dual-core SoCs from Espressif Systems with integrated WiFi and dual-mode BT module. The processor core is based on the Tensilica Xtensa LX6 32-bit Controller Processor Core.
-
-\anchor esp32_mcu
-## <a name=esp32_features> Features of ESP32 </a> &nbsp;[[TOC](#esp32_toc)]
+# Overview {#esp32_overview}
```

Comment:
In the previous documentation, each section title had a link to the table of contents. This was very helpful because the documentation is quite large and the user was able jump back to the table of contents from anywhere in the large document using these links. Would it be possible to add these links again?

Reply:
Maybe just adding
```
[Back to table of contents](#esp32_toc)
```
at the bottom of every (sub-)section just before the next section header would be enough? Syntaxwise, that should be pretty future safe regarding updates of Doxygen.